tests/fixtures/detections/disabled_rule/example_rule.yml

```diff
@@ -0,0 +1,28 @@
+AnalysisType: rule 
+Filename: example_rule.py
+DisplayName: Example Rule
+Description: Example
+Severity: High
+Threshold: 5
+RuleID: AWS.CloudTrail.MFAEnabled
+Enabled: true
+SummaryAttributes:
+  - p_log_type
+  - p_any_ip_addresses
+LogTypes:
+  - AWS.CloudTrail
+Tags:
+  - AWS Managed Rules - Security, Identity & Compliance
+  - AWS
+  - CIS
+  - SOC2
+Runbook: >
+  Find out who disabled MFA on the account.
+Reference: https://www.link-to-info.io
+Tests:
+  -
+    Name: Example Test
+    ExpectedResult: true
+    Log:
+      foo: bar
+      
```

Comment:
Nit: Looks like a minor formatting issue here. Does the new line have trailing whitespace maybe?